src/tools/wasm-merge.cpp

```diff
@@ -296,12 +296,13 @@ struct InputMergeable : public ExpressionStackWalker<InputMergeable, Visitor<Inp
       return;
     }
     curr->name = gNames[curr->name];
-    assert(curr->name.is());
-    // if this is the memory or table base, add the bump
-    if (memoryBaseGlobals.count(curr->name)) {
-      addBump(outputMergeable.totalMemorySize);
-    } else if (tableBaseGlobals.count(curr->name)) {
-      addBump(outputMergeable.totalTableSize);
+    if (curr->name.is()) {
```

Comment:
Previously the `assert(curr->name.is())` was valid, but with the removal of the tests at lines 125 above it global memory and tables are optional.